scripts/variantstore/docs/aou/workspace/CreateWriteStreamRequestIncreasedQuota.md

```diff
@@ -0,0 +1,56 @@
+## Request Additional Quota for Bulk Ingest
+BigQuery Storage API CreateWriteStream quota for small regions per minute per region for region : us-central1
+(the region for our project is us-central1 so that is the quota we needed to request an update on)
+
+Note that once the quota has been adjusted, it will not necessarily be visible as anything other than the default on this page.
```

Comment:
nit
```suggestion
**Note:** that once the quota has been adjusted, it will not necessarily be visible as anything other than the default on this page.
```